prow/cmd/pipeline/controller.go

```diff
@@ -573,7 +574,7 @@ func makePipelineGitResource(name string, refs prowjobv1.Refs, pj prowjobv1.Prow
 		ObjectMeta: pipelineMeta(name, pj),
 		Spec: pipelinev1alpha1.PipelineResourceSpec{
 			Type: pipelinev1alpha1.PipelineResourceTypeGit,
```

Comment:
this is going to be a problem need to find a way to resolve this dependency 🤔

Reply:
Using the [resource api](https://github.com/tektoncd/pipeline/blob/v0.38.0/pkg/apis/resource/v1alpha1/pipeline_resource_types.go#L70-L83) since v1beta1 refers it in the [PipelineResourceBinding](https://github.com/tektoncd/pipeline/blob/v0.36.0/pkg/apis/pipeline/v1beta1/resource_types.go#L126) as a replacement